Source/Android/app/src/main/res/values/strings.xml

```diff
@@ -117,8 +117,8 @@
     <string name="overclock_enable_description">Higher values can make variable-framerate games run at a higher framerate, requiring a powerful device. Lower values make games run at a lower framerate, increasing emulation speed, but reducing the emulated console\'s performance.</string>
     <string name="overclock_title">Emulated CPU Clock Speed</string>
     <string name="overclock_warning">WARNING: Changing this from the default (100%) WILL break games and cause glitches. Please do not report bugs that occur with a non-default clock.</string>
-    <string name="slot_a_device">Slot A Device</string>
-    <string name="slot_b_device">Slot B Device</string>
+    <string name="slot_a_device">GC Slot A Device</string>
+    <string name="slot_b_device">GC Slot B Device</string>
```

Comment:
Please write GameCube instead of GC. (Or would that make it wrap awkwardly?)

Reply:
Okay, I'll change it to that (suppose it isn't much longer than some other settings are string length wise, now that I look at it).